### PR TITLE
Added message for ArgoCD login in case of auth token expiry

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
Fixes: https://github.com/arlonproj/arlon/issues/81
As stated in this issue above, we have decided to inform the user to login into ArgoCD in case the auth token has expired

### CODE REVIEW

**Review by Rohitrajak1807 — CHANGES_REQUESTED** [review]
I'd say to avoid duplication, add it as a persistent pre run hook to the root command.
Also, failure to create a client can have other reasons as well...

File: cmd/cluster/update.go [hunk 1]
```diff
@@ -38,7 +39,7 @@ func updateClusterCommand() *cobra.Command {
 			clusterName := args[0]
 			clust, err := cluster.Get(appIf, config, argocdNs, clusterName)
 			if err != nil {
-				return fmt.Errorf("failed to get clust: %s", err)
+				return fmt.Errorf("failed to get cluster: %s", err)
```
Comment (cruizen) [inline cmd/cluster/update.go:42]:
```suggestion
				return fmt.Errorf("failed to update cluster: %s", err)
```

**Review by Rohitrajak1807 — APPROVED** [review]
seems fine... but an alternative would be to add this hook to the root command...
https://github.com/platform9/appctl/blob/master/cmd/root.go#L19
Perhaps we can refactor it later

Added the check in main.go

**Comment by ShaunakJoshi1407** [comment]
> seems fine... but an alternative would be to add this hook to the root command... https://github.com/platform9/appctl/blob/master/cmd/root.go#L19 Perhaps we can refactor it later

Added the check in main.go

**Pull request merged**